deny.toml

```diff
@@ -3,6 +3,7 @@ all-features = true
 
 [bans]
 multiple-versions = "deny"
+workspace-duplicates = "allow"
```

Comment:
What is this for?

Reply:
cargo-deny started to detect the same crates are used in the workspace but they are not defined by using workspace inheritance from version 0.15. In tonic project, a lot of crates are detected.